docs/rules.md

```diff
@@ -229,3 +230,15 @@ Check Best Practices Badge is present in README. Optionally check a certain badg
 | Input        | Required | Type       | Default | Description                                                        |
 | ------------ | -------- | ---------- | ------- | ------------------------------------------------------------------ |
 | `minPercentage` | No       | `integer`  | `null` | Minimum [Tiered Percentage](https://github.com/coreinfrastructure/best-practices-badge/blob/main/doc/api.md#tiered-percentage-in-openssf-best-practices-badge) accomplished by project. `passing=100`, `silver=200`, `gold=300`, set to `0` or `null` to disable check. |
+
+### `any-file-contents`
+Checks if the contents of at least one file in a given list match a given regular expression.
+
+| Input                    | Required | Type       | Default                             | Description                                                                                                                                                                                                                      |
+| ------------------------ | -------- | ---------- | ----------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `globsAny`               | **Yes**  | `string[]` |                                     | A list of globs to get files for. This rule passes if at least one of the files returned by the globs match the supplied string, or if no files are returned.                                                                             |
```

Comment:
Quote:
```
However, for our use case we need to check whether the content is included in at least one of the discovered files.
```

I think it doesn't make sense for the rule to pass if no files are returned. It would make sense if the rule fails if no files are returned., since we are looking for *at least there is one file has this thing there*.

That means the `fail-on-non-existent` would be `success-on-non-existent`.